lib/hyacinth/adapters/external_identifier_adapter/hyacinth_metadata.rb

```diff
@@ -2,16 +2,22 @@
 
 # Following module contains functionality to retrieve metadata
 # from the descriptive_metadata hash from DigitalObject
-class Hyacinth::Adapters::ExternalIdentifierAdapter::Datacite::Metadata
-  attr_reader :source, :descriptive_metadata
+class Hyacinth::Adapters::ExternalIdentifierAdapter::HyacinthMetadata
+  DATETIME_PARSER = Hyacinth::DigitalObject::TypeDef::DateTime.new
```

Comment:
Just a note here that I think we should be able to remove this line if we don't turn the digital object into JSON in the initialize method.